src/commands/server/start.ts

```diff
@@ -187,7 +188,18 @@ export default class Start extends Command {
       description: `Namespace for OLM catalog source to install Eclipse Che operator.
                     This parameter is used only when the installer is the 'olm'.`
     }),
-    'skip-kubernetes-health-check': skipK8sHealthCheck
+    'skip-kubernetes-health-check': skipK8sHealthCheck,
+    'workspace-engine': string({
```

Comment:
what not to add: `default: 'legacy'` 
could your remove `char: 'w'` as well. For some workspaces commands it used to specify workspacesId.

Reply:
done